examples/doxygen/block_dynamic_sparsity_pattern.cc

```diff
@@ -56,9 +56,9 @@ int main()
   DoFTools::count_dofs_per_block(dof, dofs_per_block);
 
   BlockDynamicSparsityPattern dsp(fe.n_blocks(), fe.n_blocks());
-  for (unsigned int i=0; i<fe.n_blocks(); ++i)
-    for (unsigned int j=0; j<fe.n_blocks(); ++j)
-      dsp.block(i,j).reinit(dofs_per_block[i],dofs_per_block[j]);
+  for(unsigned int i = 0; i < fe.n_blocks(); ++i)
+    for(unsigned int j = 0; j < fe.n_blocks(); ++j)
+      dsp.block(i, j).reinit(dofs_per_block[i], dofs_per_block[j]);
```

Comment:
This three lines alone make me soooooo happy! :heart: I love clang-format :heart_eyes:

Reply:
:heart_eyes_cat: